deployment/migration-assistant-solution/lib/solutions-stack.ts

```diff
@@ -188,6 +182,13 @@ export class SolutionsInfrastructureStack extends Stack {
             vpc = importVPC(this, vpcIdParameter, availabilityZonesParameter, privateSubnetIdsParameter);
         }
 
+        const cfnInitConfig: InitElement[] = [
+            InitCommand.shellCommand(`echo "export MIGRATIONS_APP_REGISTRY_ARN=${appRegistryAppARN}; export MIGRATIONS_USER_AGENT=${solutionsUserAgent}; export VPC_ID=${vpc.vpcId}; export STAGE=${stageParameter.valueAsString}" > /etc/profile.d/solutionsEnv.sh`),
```

Comment:
Nit: create a function to generate these environment variables exports

Reply:
Added